blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubOrgFolderTest.java

```diff
@@ -2,47 +2,84 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import hudson.model.User;
+import hudson.tasks.Mailer;
 import io.jenkins.blueocean.rest.impl.pipeline.PipelineBaseTest;
+import jenkins.model.Jenkins;
+import org.acegisecurity.adapters.PrincipalAcegiUserToken;
+import org.acegisecurity.context.SecurityContextHolder;
+import org.acegisecurity.userdetails.UserDetails;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.io.IOException;
 import java.util.Map;
 
 /**
  * @author Vivek Pandey
  */
 public class GithubOrgFolderTest extends PipelineBaseTest {
```

Comment:
Missing a test that the credential gets resolved, see [this](https://github.com/jenkinsci/cloudbees-folder-plugin/blob/ff0c53014c346dac3d891b150b0234c9c4c29c53/src/test/java/com/cloudbees/hudson/plugins/folder/properties/FolderCredentialsProviderTest.java#L140-L287) for inspiration but I do not thing you need to go quite so far as to actually trigger a build. I think you could get away with seeing if the credential can be looked up as SYSTEM using the standard credentials API when you add your property and if you remove the property the credential goes missing again, even though the credential is in "bob"'s user store